tools/cw/mwrap.c

```diff
@@ -490,6 +509,10 @@ int _tmain(int argc, _TCHAR *argv[])
 
     if (args.wrap_ver) tool_ver = args.wrap_ver;
 
+    if (_tcscmp(tool_ver, _T("1.2/base_b73")) == 0) {
+        remove_ipa_arguments(&new_argc, new_argv);
+    }
```

Comment:
Is there a smoother way to do this? e.g., introduce a new arg that specifies to ignore all `ipa` arguments when passing them to the compiler?

Reply:
that's probably a better approach. I will add a `-noipa` argument